java/performance_test/src/main/java/io/ray/performancetest/JobInfo.java

```diff
@@ -0,0 +1,47 @@
+package io.ray.performancetest;
+
+import com.google.common.base.Preconditions;
+import io.ray.api.call.BaseActorCreator;
+import io.ray.api.runtimecontext.NodeInfo;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class JobInfo {
```

Comment:
Do we need this class in the community version?

Reply:
removed.